playground/src/loader/loader.critical.scss

```diff
@@ -1,59 +1,17 @@
+@import 'mixins.scss';
+
 .loader {
+  z-index: 10;
   position: absolute;
-  text-align: center;
-  display: block;
-  height: 10px;
-  top: 50%;
+  top: 40%;
   left: 50%;
   transform: translateX(-50%) translateY(-50%);
   transform-origin: 50% 50%;
-  white-space: nowrap;
-  z-index: 1;
-  align-items: center;
-}
 
-.loader-dot {
-  position: relative;
-  display: inline-block;
-  height: 10px;
-  width: 10px;
-  margin: 2px;
-  border-radius: 100%;
-  background-color: color('black');
-  opacity: 0.6;
-  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
-  will-change: transform;
-  animation: loader-dots 2s infinite;
-}
-.loader.light .loader-dot {
-  background-color: color('white');
-}
-
-.loader .loader-dot {
-  &:nth-child(1) {
-    animation-delay: 0s;
-  }
-  &:nth-child(2) {
-    animation-delay: .1s;
-  }
-  &:nth-child(3) {
-    animation-delay: .2s;
+  &-fadeout {
```

Comment:
Related to https://github.com/ampproject/amp.dev/pull/4158#discussion_r450846690: this should either be `.*--fadeout` or `.fadeout`. WDYT?